mix.exs

```diff
@@ -94,7 +94,7 @@ defmodule Screens.MixProject do
       {:telemetry_metrics, "~> 1.0"},
       {:screens_config,
        git: "https://github.com/mbta/screens-config-lib.git",
-       ref: "7dbb2b9b3da0154ca9128c7c2a4ab30d18501290"},
+       ref: "3e781e39e4dd8bd63a05825534847347e1cbffa9"},
```

Comment:
Noting that this will need to be updated after merging `screens-config-lib` change into main